pyam/core.py

```diff
@@ -1163,7 +1163,13 @@ def normalize(self, inplace=False, **kwargs):
             return ret
 
     def aggregate(
-        self, variable, components=None, method="sum", recursive=False, append=False
+        self,
+        variable,
+        components=None,
+        method="sum",
+        recursive=False,
+        append=False,
+        skip_intermediate=False
```

Comment:
My hunch is that this is the line that offends stickler & Black
```suggestion
        skip_intermediate=False,
```